code/__DEFINES/greyscale.dm

```diff
@@ -0,0 +1,2 @@
+//Greyscale flags
+#define HYPERSCALE_ALLOW_GREYSCALE (1<<0)
```

Comment:
I thought kuro said no?
Also isnt this supposed to be a layer thing, ie json configed?

Reply:
This needs to be applied to the whole config since these are generated per config. since individual states inside cannot talk to eachother when one color is passed for the 'allow_greyscale' parts then all that dont have it will fail

Reply:
also kuro said sure for some specific things. The functionality can be applied largely here. What is and isnt allowed to color wheel is set on the individual items 'colorable_allowed'